mne/io/nicolet/nicolet.py

```diff
@@ -107,6 +107,15 @@ def _get_nicolet_info(fname, eog, ecg, emg, misc):
                  'ch_names': ch_names, 'description': None,
                  'buffer_size_sec': 10.})
 
+    if ch_type == 'eeg':
+        ch_coil = FIFF.FIFFV_COIL_EEG
+        ch_kind = FIFF.FIFFV_EEG_CH
+    elif ch_type == 'seeg':
+        ch_coil = FIFF.FIFFV_COIL_EEG
+        ch_kind = FIFF.FIFFV_SEEG_CH
+    else:
+        raise TypeError("Channel type not recognized. Available types are "
+                        "'eeg' and 'seeg'.")
     cal = header_info['conversion_factor'] * 1e-6
     for idx, ch_name in enumerate(ch_names):
         if ch_name in eog or idx in eog:
```

Comment:
The reason there is an or here, is that you can pass a list of indices or channel names. I'm pretty sure I copied the logic from some other reader, so I think the or should stay.